wander-embedded-sdk/src/components/button/wander-button.template.ts

```diff
@@ -64,6 +64,15 @@ export const getWanderButtonTemplateContent = ({
     transition: transform linear 50ms;
   }
 
+  .button::after {
```

Comment:
The clickable area now better match the button's visible shape:

![image](https://github.com/user-attachments/assets/e06de11f-bc00-49f8-950b-f82ff7453f85)